src/options.go

```diff
@@ -1974,6 +1979,14 @@ func parseOptions(opts *Options, allArgs []string) {
 			opts.WalkerSkip = filterNonEmpty(strings.Split(nextString(allArgs, &i, "directory names to ignore required"), ","))
 		case "--version":
 			opts.Version = true
+		case "--cpuprofile":
+			opts.CPUProfile = nextString(allArgs, &i, "file path required: cpu")
+		case "--memprofile":
+			opts.MEMProfile = nextString(allArgs, &i, "file path required: mem")
+		case "--blockprofile":
+			opts.BlockProfile = nextString(allArgs, &i, "file path required: block")
+		case "--mutexprofile":
+			opts.MutexProfile = nextString(allArgs, &i, "file path required: mutex")
```

Comment:
How about naming these with the same prefix? i.e. `--profile-cpu`, `--profile-mem`, etc.

Reply:
Done.